### PR TITLE
Remove exception raise on missing experiment

### PR DESCRIPTION
**Pull request by andela-cobiora**
@ysimonson This would prevent exceptions if experiment data is missing or incorrect.

### CODE REVIEW

File: oz/bandit/__init__.py [hunk 1]
```diff
@@ -179,12 +179,14 @@ def refresh(self):
         results = pipe.execute()
 
         if results[0] == None:
-            raise ExperimentException(self.name, "Does not exist")
-
-        self.metadata = parse_json(results[0])
-        self.choice_names = parse_json(results[1]) if results[1] != None else []
-        self.default_choice = escape.to_unicode(results[2])
-        self._choices = None
+            # no longer raising an exception, but may  need a way to alert the
```
Comment (ysimonson) [inline oz/bandit/__init__.py]: What about using `tornado.log.app_log.warning`?

File: oz/bandit/__init__.py [hunk 2]
```diff
@@ -179,12 +179,14 @@ def refresh(self):
...
+            # occurrence
+            pass
+        else:
+            self.metadata = parse_json(results[0])
```
Comment (ysimonson) [inline oz/bandit/__init__.py]: None of these variables will be set now, causing exceptions down the line

**Comment by andela-cobiora** [comment]
@ysimonson refactor is up using `tornado.log.app_log.warning`. I QA'ed by making sure that all the related variables remain correctly handled in the event of a bad experiment. Help review thanks.

**Comment by ysimonson** [comment]
Merged, but accidentally did it to master! I'm going to revert that, and re-merge to develop.

**Pull request merged**